spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -115,21 +115,21 @@ module.exports = function (provider, setup) {
       var paused = false;
       var messageCount = 0;
       dispatch.on('onMessage', function (msg) {
-        if (!('data' in msg)) {
+        if (msg && !(msg.hasOwnProperty('data'))) {
```

Comment:
msg was null when this was receiving onDisconnect, now that I've commented that out I can remove this check.